src/firebase.js

```diff
@@ -0,0 +1,36 @@
+import firebase from "firebase/app";
+import "firebase/auth";
+import "firebase/firestore";
+
+// <!-- The core Firebase JS SDK is always required and must be listed first -->
+// <script src="https://www.gstatic.com/firebasejs/7.23.0/firebase-app.js"></script>
+
+// <!-- TODO: Add SDKs for Firebase products that you want to use
+//      https://firebase.google.com/docs/web/setup#available-libraries -->
+// <script src="https://www.gstatic.com/firebasejs/7.23.0/firebase-analytics.js">
```

Comment:
This part is only required for a non-React app, so you can safely remove these comments :)

Reply:
Comments removed